litmus-portal/frontend/src/components/ReliabilityScore/styles.ts

```diff
@@ -52,6 +52,162 @@ const useStyles = makeStyles((theme) => ({
   buttonOutlineText: {
     paddingLeft: theme.spacing(1.5),
   },
+  modalContainer: {
+    marginLeft: '15rem',
+    marginRight: '15rem',
+    marginBottom: '5rem',
+    marginTop: '2rem',
+    paddingLeft: 30,
+    paddingRight: 30,
```

Comment:
can we take the padding properties from theme data

Reply:
@arkajyotiMukherjee 

- [x] Fixed